android/quest/src/main/java/org/smartregister/fhircore/quest/ui/patient/register/PatientRegisterActivity.kt

```diff
@@ -48,17 +54,26 @@ class PatientRegisterActivity : BaseRegisterActivity() {
 
   override fun bottomNavigationMenuOptions(): List<NavigationMenuOption> {
     return listOf(
-      NavigationMenuOption(
-        id = R.id.menu_item_clients,
-        title = getString(R.string.menu_clients),
-        iconResource = ContextCompat.getDrawable(this, R.drawable.ic_users)!!
-      ),
-      NavigationMenuOption(
-        id = R.id.menu_item_settings,
-        title = getString(R.string.menu_settings),
-        iconResource = ContextCompat.getDrawable(this, R.drawable.ic_settings)!!
+        NavigationMenuOption(
```

Comment:
cool this looks good, what's the strategy to make this configurable? could load a list of menu items from a config then loop through theme here adding `NavigationMenuOption`s to the list

Reply:
Line 62 does this where it loads nav from config.. however, first two navigation options are for settings/register list which is not dynamic and opens a specific activity (Settings screen/Client Register). This is default app behaviour and we have not devised any strategy yet to make this configurable